examples/amp-story/quiz.html

```diff
@@ -64,7 +64,8 @@ <h1 id="cat-facts-title">Cat Facts!</h1>
       </amp-story-grid-layer>
       <amp-story-grid-layer>
         <amp-story-quiz
-          id='cat-question-appear'>
+          id='cat-question-appear'
+          endpoint="http://localhost:8000">
```

Comment:
We can revisit later but I'd really like us to consider both approaches to providing configuration to a quiz before we launch, parameters vs JSON

Reply:
We can sync about this option a bit more tomorrow offline